app/models/invoice.rb

```diff
@@ -21,6 +21,14 @@ def self.incomplete_invoices
       .order(:created_at)
   end
 
+  def self.best_day
```

Comment:
This actually looks similar to a method Lance and I are working on. Will be interested to see what ours turns out as